modules/govuk/manifests/apps/feedback.pp

```diff
@@ -140,4 +150,9 @@
     varname => 'GOVUK_NOTIFY_ACCESSIBLE_FORMAT_REQUEST_REPLY_TO_ID',
     value   => $govuk_notify_accessible_format_request_reply_to_id,
   }
+
+  govuk::app::envvar::redis { $app_name:
+    host => $redis_host,
+    port => $redis_port,
```

Comment:
I think these [will default to 127.0.0.1 / 6379](https://github.com/alphagov/govuk-puppet/blob/main/modules/govuk/manifests/app/envvar/redis.pp#L24-L25) in non-integration environments. Is that going to cause trouble?

You could wrap this with an `if $redis_host != undef {` just to be sure?

Reply:
I... don't know - hopefully by the time it gets to non-integration environments we'll have a better change here (because we don't want rate-limiting cache stuff pumped into the redis instance everything else shares), but yeah, no harm in being cautious and putting in that wrapper.

Reply:
>  by the time it gets to non-integration environments

Always better to be careful with these things - if it gets merged to main, you should probably assume someone will deploy it everywhere. Thanks for the wrapper :)